jbi/models.py

```diff
@@ -330,20 +329,11 @@ class Config:
 
     def map_as_jira_comment(self):
         """Extract comment from Webhook Event"""
-        comment: BugzillaWebhookComment = self.bug.comment
         commenter: BugzillaWebhookUser = self.event.user
-        comment_body: str = comment.body
-
-        if comment.is_private:
-            bug_comments = get_bugzilla().get_comments([self.bug.id])
-            comment_list = bug_comments["bugs"][str(self.bug.id)]["comments"]
-            matching_comments = [c for c in comment_list if c["id"] == comment.id]
-            if len(matching_comments) != 1:
-                return None
-            comment_body = matching_comments[0]["text"]
-
-        body = f"*({commenter.login})* commented: \n{{quote}}{comment_body}{{quote}}"
-        return body
+        comment: BugzillaWebhookComment = self.bug.comment
+        if comment is None:
+            return None
+        return f"*({commenter.login})* commented: \n{{quote}}{comment.body}{{quote}}"
```

Comment:
Do not need to account for private comments anymore? I wonder if we should raise a `ValueError` if `comment.is_private`.

Reply:
Comment is `None` if comment is private. I'll add a comment, but I didn't want to change the previous behavior

Reply:
I'm all for returning `None`, but I was wondering if we still needed the guard. What happens if we try to access `comment.body` on a private or nonexistent `webhookrequest.bug.comment`?